src/altinn-app-frontend/src/components/presentation/NavigationButtons.test.tsx

```diff
@@ -93,6 +93,7 @@ describe('NavigationButton', () => {
     render(
       <Provider store={mockStore}>
         <NavigationButtons
+          type={'NavigationButtons'}
```

Comment:
Seems a bit redundant to have to specify the type on the component in tests - any easy way to avoid having to do this?

Reply:
Yeah, you're right. I added `Omit<..., 'type'>` to these prop types instead.